hatohol.spec.in

```diff
@@ -67,55 +68,50 @@ Group: Applications/System
 Requires: hatohol-lib-arm = %{version}
 
 %description arm-zabbix
-arm-zabbix collects monitoring data from a Zabbix server and sends
+hatohol-arm-zabbix collects monitoring data from a Zabbix server and sends
 it to hatohol via HAPI.
 
 %package arm-ceilometer
-Summary: Arm for Ceilometer.
+Summary: Arm for OpenStack Ceilometer.
 Group: Applications/System
 Requires: hatohol-lib-arm = %{version}
 
 %description arm-ceilometer
-arm-ceilometer collects monitoring data from a Ceilometer and sends
-it to hatohol via HAPI.
+hatohol-arm-ceilometer collects monitoring data from a OpenStack Ceilometer
+and sends it to hatohol via HAPI.
 
 %package lib-common
 Summary: Library files for Hatohol
 Group: Applications/System
 
 %description lib-common
-Common libraries for hatohol and arm plugins.
+Common libraries for hatohol and arms.
 
 %package lib-arm
-Summary: Library files for Hatohol Arm
+Summary: Library files for Hatohol arms.
 Group: Applications/System
 Requires: hatohol-lib-common = %{version}
 
 %description lib-arm
-Libraries for arm.
+Libraries for arms.
 
 %package web
-Summary: A web UI web of Hatohol.
+Summary: Hatohol web frontend.
 Group: Applications/System
 Requires: MySQL-python >= 1.2.3
-%if %is_el6
 Requires: Django >= 1.5.3
-%endif
-%if %is_el7
-Requires: python-django
```

Comment:
bynnchapu san

I confirm this problem.
When I didn't add Hatohol repository and local install Hatohol 14.12 from rpm,
Hatohol passed minimal test. Then, python-django-1.6.8-1.el7.noarch was installed in this machine. 

So, I think we need not to rebuild Django package for CentOS7.